Src/IdiomsUnitTest/ConstructorInitializedMemberAssertionTest.cs

```diff
@@ -705,5 +802,11 @@ int IEqualityComparer.GetHashCode(object obj)
                 throw new NotImplementedException();
             }
         }
+
+        enum TestEnum { none = 0, one, two, three };
+
+        enum TestDefaultOnlyEnum { none = 0 };
+
+        enum TestSingleNonDefaultEnum { none = 1 };
```

Comment:
These are used as type arguments in the arrange phase of the newly added [unit tests](https://github.com/Tungsten78/AutoFixture/blob/905a836fe4248d40447e11b758ccfd01ae4f1ae6/Src/IdiomsUnitTest/ConstructorInitializedMemberAssertionTest.cs#L535).